src/ranged.h

```diff
@@ -44,6 +44,7 @@ trajectory mode_spell( avatar &you, spell &casting, bool no_fail, bool no_mana )
 } // namespace target_handler
 
 void practice_archery_proficiency( player &p, const item &relevant );
+void mod_stamina_archery( player &p, const item &relevant );
```

Comment:
To have functions declared in only a `.cpp` file (e.g. to make these entries not required), you can add `static` before the return type.